### PR TITLE
Removed note from Tools and utitilies guide (Sd/content/df 237)

### PR DESCRIPTION
**Pull request by sheet20**
## What Changed?

DF-237: Removed notes from tools and utilities guide. Pls merge with develop.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/epas/12/epas_compat_bip_guide/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 navTitle: Built-in Package Guide
-title: "Database Compatibility for Oracle Developers Built-in Package Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_bip_guide/index.mdx:3]: In this instance plural is appropriate. I would not change it.

File: product_docs/docs/epas/12/epas_compat_bip_guide/index.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+title: "Database Compatibility for Oracle Developer Built-in Package Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_bip_guide/index.mdx:3]: make Developer plural

File: product_docs/docs/epas/12/epas_compat_cat_views/index.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 navTitle: Catalog Views Guide
-title: "Database Compatibility for Oracle Developers Catalog Views Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_cat_views/index.mdx:3]: In this title it makes sense to keep Developers plural.

File: product_docs/docs/epas/12/epas_compat_cat_views/index.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+title: "Database Compatibility for Oracle Developer Catalog Views Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_cat_views/index.mdx:3]: same comment

File: product_docs/docs/epas/13/epas_compat_ora_dev_guide/index.mdx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 navTitle: User Guide
-title: "Database Compatibility for Oracle Developer's Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/13/epas_compat_ora_dev_guide/index.mdx:3]: make plural not possessive

**Comment by sheet20** [comment]
You can put your comments and merge the changes, later I'll incorporate it.

**Review by drothery-edb — CHANGES_REQUESTED** [review]
When using the "Database Compatibility for Oracle Developer" in the title, Developer should be plural but not possessive

**Comment by sheet20** [comment]
The changes is been addressed with PR #1451.

**Pull request closed without merging**